src/pages/Flow/UpcomingRuns-Tile.vue

```diff
@@ -178,8 +190,13 @@ export default {
       },
       loadingKey: 'loadingKey',
       pollInterval: 10000,
-      fetchPolicy: 'no-cache',
-      update: data => data?.flow_run || []
+      update: data => {
+        return (data?.flow_run || []).map(run => {
+          run.cacheInvalidation = new Date().getTime()
```

Comment:
nitpick: make `new Date().getTime()` a const and use the same date for every flow run.

Reply:
🧠